pkg/infra/game/message/message.go

```diff
@@ -48,3 +48,21 @@ func (s StartFight) sealedInform() {
 func (s StartFight) sealedFightInform() {
 	panic("implement me")
 }
+
+type ArrayInfo struct {
```

Comment:
Could this be moved into a separate file within the same package or even a sub-package just to keep things tidy?